# HG changeset patch
# User xunxun1982
# Date 1411495294 -28800
#      Wed Sep 24 02:01:34 2014 +0800
# Branch GECKO340b9_2014111321_RELBRANCH
# Node ID e1965d3e6ebba490d2bba3bb9568d6a0737b1912
# Parent  5fb3dfee1009ca8f02347b673ded2a4b4b584210
slim-little

diff -r 5fb3dfee1009 -r e1965d3e6ebb configure.in
--- a/configure.in	Sun Nov 16 16:07:41 2014 +0800
+++ b/configure.in	Wed Sep 24 02:01:34 2014 +0800
@@ -1635,7 +1635,7 @@
 dnl ========================================================
 dnl SPS Profiler
 dnl ========================================================
-MOZ_ENABLE_PROFILER_SPS=1
+MOZ_ENABLE_PROFILER_SPS=
 
 case "${OS_TARGET}" in
 Android)
